Rakefile

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env rake
+
+desc "Update vendor/cookbooks"
+task :vendor do
+  puts "Vendoring cookbooks"
+  `rm -rf vendor`
+  `berks update`
+  `berks vendor vendor/cookbooks`
+end
+
+desc "Package cookbooks into a chef-solo tarball"
+task :package => :vendor do
+  `mkdir -p vendor/cookbooks/conjur`
+  `cp -r metadata.rb Berksfile Berksfile.lock CHANGELOG.md chefignore README.md attributes recipes spec vendor/cookbooks/conjur`
+  version=`git describe --tags --abbrev=0`.strip
```

Comment:
Say you tag 'v0.2.3'. `git describe --tags --abbrev=0` will correctly return 'v0.2.3', as will `git describe --tags`. Now you add a few commits; say you're at `8c94a049458f614a031537df759b5b0621f75f0d`. Now `git describe --tags --abbrev=0` will _still_ return 'v0.2.3', even though the code doesn't actually correspond to that tag; `git describe --tags` will say something like 'v0.2.3-8c94a049458f', which makes it clear that it's code based on v0.2.3, but not exactly at the tag.